hs_communities/management/commands/check_czo_groups.py

```diff
@@ -0,0 +1,208 @@
+
+                    if owners.count() == 0:
+                        print("  No owners for {} {}:".format(r.short_id,
+                                                              r.title.encode('ascii',
+                                                                             'ignore')))
+                    if owners.count() == 2:
+                        print("  Too many owners for {} {}:".format(r.short_id,
+                                                                      r.title.encode('ascii',
+                                                                                     'ignore')))
+                        for o in owners:
+                            print("    {}".format(o.username))
+
+                    groups = Group.objects.filter(g2grp__resource=r)
+
+                    if groups.count() == 0:
+                        print("  No groups for {} {}:".format(r.short_id,
+                                                              r.title.encode('ascii', 'ignore')))
+                    elif groups.count() > 1:
+                        print("  Extra groups for {} {}:".format(r.short_id,
+                                                                 r.title.encode('ascii', 'ignore')))
+                        for g in groups:
+                            print("    {}".format(g.name))
+                for r in user_resources - group_resources:
+                    # print("resource {} is owned but not in czo_national group"
+                    #       .format(r.short_id, r.title.encode('ascii', 'ignore')))
+                    title = r.title
+                    prefix = title.split(' ')[0]
+                    # print("{} {} title prefix is {}".format(r.short_id,
+                    #                                         r.title.encode('ascii', 'ignore'),
+                    #                                         prefix))
+                    if prefix in prefixes:
+                        newgroupname = prefixes[prefix][1]
+
+                        # print("  prefix {} is for group {}".format(prefix, newgroupname))
+                        newgroup = Group.objects.get(name=newgroupname)
+                        if newgroup not in Group.objects.filter(g2grp__resource=r):
+                            print("  resource {} {} should be in group {}"
+                                  .format(r.short_id,
+                                          r.title.encode('ascii', 'ignore'),
+                                          newgroupname))
+                    else:
+                        print("  prefix {} is unknown".format(prefix))
+
+
+def is_equal_to_as_set(l1, l2):
+    """ return true if two lists contain the same content
+    :param l1: first list
+    :param l2: second list
+    :return: whether lists match
+    """
+    # Note specifically that set(l1) == set(l2) does not work as expected.
+    return len(
```

Comment:
Something like this may be a little easier to understand, also, l1 and l2 are already sets when passes as parameters
`return len(temp1.symmetric_difference(temp2)) == 0`

Reply:
Fixed. I need to be able to act on lists in other commands. Keeping consistent.